### ISSUE TITLE
102632488 UserID Profile display has errors (Eric)

### ISSUE THREAD

**Issue by FreeUKGenIssues** [FreeUKGen/FreeUKRegProductIssues#1058]
Issue reported by **REGManager** at 2018-04-02 21:01:28 UTC
Time: 2018-04-02T20:56:34+00:00
Session ID: 4f83592c284978ecaa9b4126b3cb9b7c
Problem Page URL: [/userid_details/563b5356f493fd25780002f5](/userid_details/563b5356f493fd25780002f5)
Previous Page URL: [https://www.freereg.org.uk/userid_details/563b5356f493fd25780002f5/edit](https://www.freereg.org.uk/userid_details/563b5356f493fd25780002f5/edit)
Reported Issue:
The display of the UserID profile has changed and  there are some spelling errors in the field names.  See attached.
Also, "Active" yes or no is not displayed.
Eric
http://www13.freereg.org.uk/uploads/feedback/screenshot/5ac29a28f493fd69278acde9/Capture.JPG    

![Screenshot](http://www.freereg.org.uk/uploads/feedback/screenshot/5ac29a28f493fd69278acde9/Capture.JPG)

**Comment by Sherlock21** [comment]
Even more surprising: On T2 and T3, the spellings are correct, but the Active/ Not Active field is not displayed.
So how did this get changed in production before testing in one or other of the test areas if its been done "formaly"?

**Comment by Captainkirkdawson** [comment]
All servers including test2 and test3 are all running the same profile modules. These were changed in story  #1310 which went through normal deployment practices and was deployed into production on 7 March.
Pity that spelling mistakes were not identified at that time as part of testing.

**Comment by Captainkirkdawson** [comment]
@edickens  @Sherlock21  Spelling corrected. Active added. Committed to master. Available for testing on test3

**Comment by Sherlock21** [comment]
I have checked my own profile in T3 and all the spelling seems correct.

As an aside, I tried to edit my profile to correct: Previous Syndicate ( to Norfolk, Julie Harold) but there is no means of doing that - so how does that get corrected please?

**Comment by Captainkirkdawson** [comment]
There is no way. Why do you want to do that in the test environment?

**Comment by Sherlock21** [comment]
I was just checking out the spelling of each item on Profile page - as your msg requested.
I picked MY profile as that was handy to access, and noticed that my Previous Profile says WRY, whereas it was NFK. So I though I would see if it was editable so I could correct it.
I: agree that its nothing to do with the task of "testing", as I say in my report.

But keeping things accurate helps  the job work better I always find.
If I had been able to correct my Previous Syndicate there, in T3, I would have gone and done same in Production. - If I had messed it up, then better to do it in t3 rather than in Prod.
